assets/queries/openAPI/general/undefined_security_scope_global_security/query.rego

```diff
@@ -45,5 +50,16 @@ check_schemes(doc, secFieldSchemes) = secFieldScheme {
 
 	arr := [x | _ := secScheme.flows[flowKey].scopes[scopeName]; scopeName == secScope; x := secScope]
 
+	count(arr) == 0
+} else = secFieldScheme {
+	version == "2.0"
+	secFieldSecurityScheme := secFieldSchemes[secFieldScheme]
+	secScheme := doc.components.securityDefinitions[scheme]
+	secScheme.type == "oauth2"
+
+	secScope := secFieldSecurityScheme[_]
+
+	arr := [x | _ := secScheme.scopes[scopeName]; scopeName == secScope; x := secScope]
+
 	count(arr) == 0
 }
```

Comment:
I think you could refactor this to using `secScheme` as parameters, since the check is almost the same, besides from where you get `secScheme`, no?